src/atomate2/vasp/sets/core.py

```diff
@@ -543,6 +547,135 @@ def incar_updates(self) -> dict:
         return updates
 
 
+@dataclass
+class MVLGWSetGenerator(VaspInputGenerator):
```

Comment:
In the latest version of atomate2, we have moved to using the input sets defined in pymatgen. Can you remove this definition and import the pymatgen versions instead (hopefully they should just be a drop in replacement). If you found any bugs etc in the MVL input sets then can you submit a PR to pymatgen.